README.md

```diff
@@ -1,70 +1,82 @@
-<h1 align="center">Welcome to Certificate-Generator</h1> 
-<br>
-
 <div align="center">
-  <img src="https://fossnaija.com/wp-content/uploads/2022/09/fossnaija.com-hacktoberfest-2022-1024x455.png" alt="hacktoberfest22">
-</div>
+<h1 align="center"> <img src= "https://octodex.github.com/images/original.png" width= "40" /> HacktoberFest 2022 <img src= "https://octodex.github.com/images/original.png" width= "40" /> </h1>
 
-<br>
 
-<div align="center">
-  <img src="https://forthebadge.com/images/badges/for-you.svg" />
-  <img src="https://forthebadge.com/images/badges/made-with-markdown.svg" alt="css">
-  <img src="https://forthebadge.com/images/badges/made-with-javascript.svg" alt="js">
-  <img src="https://forthebadge.com/images/badges/open-source.svg" alt="os">
-  <img src="https://forthebadge.com/images/badges/check-it-out.svg" alt="check">
-  <img src="" alt="">
-  <img src="https://forthebadge.com/images/badges/built-by-developers.svg" />
-</div>
+![image](https://user-images.githubusercontent.com/54996800/192564253-f3373ecc-bc86-4c13-a0bd-551b9c23f788.png)
 
-## Setup Steps
+[![GitHub issues](https://img.shields.io/github/issues/vedant-jain03/certificate-generator?style=for-the-badge&color=important)](https://github.com/vedant-jain03/certificate-generator/issues)
+[![GitHub forks](https://img.shields.io/github/forks/vedant-jain03/certificate-generator?style=for-the-badge&color=blue)](https://github.com/vedant-jain03/certificate-generator/fork)
+[![GitHub stars](https://img.shields.io/github/stars/vedant-jain03/certificate-generator?style=for-the-badge&color=yellow)](https://github.com/vedant-jain03/certificate-generator/stargazers)
+</div>
+</br>  
 
-- Go to directory
-```
-$ cd certificate-generator
-```
-- Install Dependencies
-```
-$ npm install
-```
-- Rename .env.example to .env and add your Firebase api key
-```
-$ mv .env.example .env
-```
-- Start LocalHost Server
-```
-$ npm run start
-```
+<h2> What is Hacktoberfest? </h2>
```

Comment:
It is not needed, you can just refer hacktoberfest website, something like learn more about hacktoberfest here!